mappings/net/minecraft/client/util/InputUtil.mapping

```diff
@@ -1,39 +0,0 @@
-CLASS ctu net/minecraft/client/util/InputUtil
```

Comment:
Where did this go? I assume it was a class being renamed, but you should clarify this in the PR description.

Reply:
`net.minecraft.client.input.Input`